src/goto-programs/link_to_library.cpp

```diff
@@ -25,24 +79,27 @@ Author: Daniel Kroening, kroening@kroening.com
 void link_to_library(
   goto_modelt &goto_model,
   message_handlert &message_handler,
-  const std::function<
-    void(const std::set<irep_idt> &, symbol_tablet &, message_handlert &)>
-    &library)
+  const std::function<void(
+    const std::set<irep_idt> &,
+    const symbol_tablet &,
+    symbol_tablet &,
```

Comment:
Can we document the two symbol table parameters and what they are supposed to be set to?

Reply:
Documentation added.